app/views/workflow_executions/_files.html.erb

```diff
@@ -1,60 +1,35 @@
-<div class="relative overflow-x-auto" data-turbo-temporary>
+<div class="relative overflow-x-auto">
   <table
     id="files-table"
-    class="
-      w-full
-      text-sm
-      text-left
-      rtl:text-right
-      text-slate-500
-      dark:text-slate-400
-      whitespace-nowrap
-    "
+    class="w-full text-sm text-left table-auto  rtl:text-right text-slate-500 dark:text-slate-400 whitespace-nowrap"
```

Comment:
Extra space in the classes

Reply:
Appears to be added by the TailwindCSS formatter